extension/js/common/message-renderer.ts

```diff
@@ -468,6 +468,16 @@ export class MessageRenderer {
     };
   };
 
+  private clipMessageIfLimitExceeds = (decryptedContent: string) => {
+    const maxDecryptedContentLength = 100000;
+    const base64InlineImageRegex = /<img src="data:image\/[^;]+;base64,[^"]+" name="(\w+\.\w+)" title="[^"]+">/;
```

Comment:
This regex catches only first image, we should use `/g` flag for parsing all images.

Reply:
Thank you. I will be more attentive to the regex flag next time I encounter it.